manage_arkime/core/capacity_planning.py

```diff
@@ -350,18 +351,79 @@ def get_os_domain_plan(expected_traffic: float, spi_days: int, replicas: int, nu
 
     return OSDomainPlan(data_node_plan, master_node_plan)
 
+class InvalidCidr(Exception):
+    def __init__(self, cidr_str: str):
+        super().__init__(f"The string {cidr_str} is not a valid CIDR block")
+
+class Cidr:
+    def __init__(self, cidr_str: str):
+        self._validate_cidr(cidr_str)
+
+        self.block = cidr_str
+        self.prefix = cidr_str.split("/")[0]
+        self.mask = cidr_str.split("/")[1]
+
+    def _validate_cidr(self, cidr_str: str ):
+        overall_form = re.compile("^[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}/[0-9]{1,2}$")
+        if not overall_form.match(cidr_str):
+            raise InvalidCidr(cidr_str)
+        
+        prefix_portion = cidr_str.split("/")[0]
+        prefix_portions_correct = list(map(lambda x: int(x) >=0 and int(x) <= 255, prefix_portion.split(".")))
+        if not all(prefix_portions_correct):
+            raise InvalidCidr(cidr_str)
+        
+        mask_portion = cidr_str.split("/")[1]
+        mask_portion_correct = int(mask_portion) >= 0 and int(mask_portion) <= 32
```

Comment:
I would think we shouldn't allow a /32, or maybe even a /28 because of the above. Maybe this is checked somewhere else.